src/main/java/org/spongepowered/api/service/scheduler/Task.java

```diff
@@ -48,21 +47,26 @@
     PluginContainer getOwner();
 
     /**
-     * Gets the delay that the task was scheduled to run after.
+     * Gets the delay that the task was scheduled to run after. A delay of 0
+     * represents that the task will start immediately.
      *
-     * @return The delay (offset) in the scale of the time unit applied if asynchronous, otherwise raw synchronous ticks
+     * @return The delay (offset) in the scale of the time unit applied if
```

Comment:
Should probably use the same defaults as the builder, maybe provide `getDelay(TimeUnit)` additionally?